rs/sns/governance/src/types.rs

```diff
@@ -3335,83 +3621,185 @@ pub(crate) mod tests {
         }
     }
 
-    impl NeuronParameters {
-        fn validate_default() -> Self {
+    impl NeuronRecipe {
+        fn validate_default_direct_participant() -> Self {
             Self {
                 controller: Some(*TEST_USER1_PRINCIPAL),
-                hotkey: Some(*TEST_USER2_PRINCIPAL),
+                neuron_id: Some(NeuronId::new_test_neuron_id(0)),
                 stake_e8s: Some(E8S_PER_TOKEN),
                 dissolve_delay_seconds: Some(3 * ONE_MONTH_SECONDS),
-                source_nns_neuron_id: None,
+                followees: Some(NeuronIds::from(vec![NeuronId::new_test_neuron_id(1)])),
+                participant: Some(Participant::Direct(neuron_recipe::Direct {})),
+            }
+        }
+
+        fn validate_default_neurons_fund() -> Self {
+            Self {
+                controller: Some(PrincipalId::from(CanisterId::from_u64(1185))), // NNS Governance
```

Comment:
This looks weird. If this is indeed NNS Gov, why not use a constant?

Reply:
Done